apps/freenet-microblogging/Makefile

```diff
@@ -16,7 +17,7 @@ webapp:
 	cd apps/freenet-microblogging
 	cd $(WEB_DIR)
 
-	${CARGO_TARGET_DIR}/debug/ldt build
+	CARGO_TARGET_DIR="${CARGO_TARGET_DIR}" ${CARGO_TARGET_DIR}/debug/ldt build
```

Comment:
minor grip, can we split this in two lines so is easier to see?
not sure long term this script will be maintained but at least for now that would be easier to see

Reply:
I now just export the necessary env var, rather than declare it on every invocation of `ldt`.  works fine and is cleaner.